### DISCUSSION TITLE
Welcome DevCycle to the Student Developer Pack!

### DISCUSSION THREAD

**Discussion by morganersery1** [community/community#60269] [GitHub Education]
### Hi everyone,

![DevCycle New Partner SDP copy](https://github.com/community/community/assets/107881423/1ec6db45-d6ab-4dc0-84c6-e13fe883c957)

### Feature Flag Management Made Easy
DevCycle is a feature flag platform built for teams of any size. Easily create, rollout, and cleanup feature flags without disrupting your workflow. As the newest partner to join the pack, DevCycle’s generous offer includes 1 Free Year on their Starter Plan to all Verified Students. This includes unlimited seats, unlimited feature flags, and unlimited usage. 

Starting today, verified students can leverage DevCycle's cutting-edge feature flag management capabilities at no cost. With DevCycle, you will be able to toggle features on or off seamlessly, control feature rollouts, perform A/B testing, and most importantly, prevent or mitigate issues that might affect your user experience.

### Getting Started
Wondering how to make the most of this offer? [View this quick video](https://www.youtube.com/watch?v=bZD-pyKGwR4) that walks you through adding your first feature flag in just three easy steps, then [redeem your benefit](https://education.github.com/students) today.

[https://www.youtube.com/watch?v=bZD-pyKGwR4](https://www.youtube.com/watch?v=bZD-pyKGwR4)
<img width="1268" alt="Screenshot 2023-07-05 at 5 18 20 PM" src="https://github.com/community/community/assets/107881423/af612d92-4d46-4202-a44c-9b71b0195f25">

**Comment by andrewdmaclean** [comment]
So excited to join the Student Developer Pack 🎉  

If anyone has questions, please feel free to reach out here or on our Discord Community at https://bit.ly/devcycle-discord 🙏

**Comment by nikolasleblanc** [comment]
This is awesome. So excited for students to get their hands on DevCycle. Can’t wait to see what they do with them!
